File: Python/deal_with.py



import re

source_file="simpleweixiao.txt"
dest_file="simpleweixiao_OK.txt"
f=open(source_file,'r')
t=open(dest_file,'w')

for line in f.readlines():
	if(len(line)!=0):
		print(line)
		t.write(line)
f.close()
t.close()

File: Perl/deal_with.pl


use strict;

my $source_file="simpleweixiao.txt";  #源文件名
my $dest_file="simpleweixiao_OK.txt"; #目标文件名
my @dest=();
my $one;
my $number= 1;

open(SOURCE_FILE,"<$source_file") or die;
while(<SOURCE_FILE>)
{

s/[\d]|[\.\,\%\*]//g;      # let them to be space'_' 
push(@dest,$_);

}
close(SOURCE_FILE);

open(DEST,">$dest_file") or die;

foreach $one (@dest){
unless($one=~/^\s$/)  #剔除空白行
{
print DEST "$number $one";
$number++;
}
}

close(DEST);
